site/content/docs/developer/building-zap-with-intellij-idea.md

```diff
@@ -79,6 +79,51 @@ Note that the add-ons in _zap-extensions_ all have their own set of tasks in add
 
 ## Tips and Tricks
 
+### Connecting to a session database
+
+You can use the Database tab in IntelliJ IDEA to connect to a persisted ZAP session.
+By default, ZAP uses [HSQLDB](https://hsqldb.org/) to store its session data.
+There is also experimental support for MySQL and PostgreSQL but we won't be discussing that here.
+
+Since ZAP locks its session file, it is only possible to view or modify the database when ZAP doesn't have the session open.
+By default, ZAP stores its session files under `session/` in its [default directory](/faq/what-is-the-default-directory-that-zap-uses/).
+
+To open a session database in IDEA, click on the Database tab, and add a new HSQLDB data source.
+
+![Adding a new HSQLDB Data Source to IntelliJ IDEA](/img/docs/developer/intellij-db-add.png)
+
+Then, in the window that pops up:
+- specify a `Name` for the connection
+- enter or select the `Path` to the directory containing your session file
+- set the `Database` name equal to the ZAP session file name
+- set the `User` equal to "`sa`".
```

Comment:
Should be consistent in use (or not) of periods in bullets.

I’d suggest just deleting this one unless you’re going to use leading capitals and actual sentences.